src/uct/ib/rc/base/rc_ep.c

```diff
@@ -216,7 +211,7 @@ static UCS_CLASS_CLEANUP_FUNC(uct_rc_ep_t)
     uct_rc_ep_tag_qp_destroy(self);
```

Comment:
should not we call ` ucs_derived_of(iface->super.ops, uct_rc_iface_ops_t)->destroy_qp` for tag qp as well?
some RNDVs may be in progress

Reply:
yes